cimetrics/upload_complete.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import cimetrics.upload
```

Comment:
This is pretty short, but it still seems better to offer one than for everybody to have a copy.